src/cpp/session/resources/themes/compile-themes.R

```diff
@@ -1067,6 +1067,18 @@
       replace = operatorBgColor
    )
    
+   ## Provide a color for invisible characters if none is defined.
```

Comment:
The tmTheme -> ace conversion code should always generate an invisible color, it's just not being used right now (there's a default if one isn't set, but it seems like most themes have one set). I think to fix this we should either add 
```
.%cssclass% {
   color: %invisible%
}
```
to the ace_theme_template.css file (here: https://github.com/rstudio/rstudio/blob/master/src/cpp/session/resources/templates/ace_theme_template.css)